src/utils/safeTransactions.ts

```
@@ -4,20 +4,33 @@ import {
   SafeModuleTransactionWithTransfersResponse,
   SafeMultisigTransactionWithTransfersResponse,
+  SafeDelegateListResponse,
 } from '@safe-global/api-kit';
 import { SafeMultisigTransactionResponse } from '@safe-global/safe-core-sdk-types';
 import { Address, decodeFunctionData, formatEther, formatUnits } from 'viem';
 import { erc20Abi, erc4626Abi, erc721Abi } from 'viem';
 import { truncateEthereumAddress } from './blockchain';
 import { web3 } from '@hoprnet/hopr-sdk';
 import { wxHOPR_WRAPPER_SMART_CONTRACT_ADDRESS } from '../../config';
+import { createNewConfigTx } from '../pages/staking-hub/onboarding/step6/0configureModule';
+import { safe } from 'wagmi/connectors';
```

ESLint

[error] 'safe' is defined but never used.

(@typescript-eslint/no-unused-vars)